### PR TITLE
Update docs to include ping script tracking

### PR DESCRIPTION
**Pull request by chrissvo**
Fixes: https://github.com/mgmco/bugs/issues/2169

Changes proposed in this pull request:
 * TSIA

cc @mgmco/developers

### CODE REVIEW

**Comment by chrissvo** [comment]
@mtimofiiv I wasn't sure, in the new situation where the client sets the cookie + uses the ping script. Does the name of the cookie have to be specified somewhere?
```html
<script src="https://tracking.membergetmember.co/scripts/ping.js"></script>
<script>
  (function() {
    var MGMbuyerInfo = {
      email: 'client@email.com', // Required
      name: 'Your new client\'s name',
      bank_account: 'Your new client\'s bank account',
      state: 'confirmed',
      customer_id: 'Your new client\'s customer ID',
      // Insert other parameters if needed
    };

    window.SocialShopSale = MGMbuyerInfo;
    window.SocialShopPing.refresh();
  })();
</script>
```

**Comment by harianus** [comment]
The `\'s` looks a bit strange. Why not `name: "Your new client's name",`?

**Comment by harianus** [comment]
It looks like you are still going for API tracking mainly? I would say the getting started page of tracking should have the focus on our ping script right?

File: _sales_tracking/callbacks.md [hunk 1]
```diff
@@ -1,102 +1,21 @@
...
-  var request = new RestRequest(Method.POST);
-  IRestResponse response = client.Execute(request);
-  ~~~
-  {: title="C#"}
-  ~~~python
-  import http.client
+---
 
-  conn = http.client.HTTPConnection("api.membergetmember.co")
+On 2 occasions your API will have to make a callback to our API:
```
Comment (harianus) [inline _sales_tracking/callbacks.md]: It is not their API when going for option 2 of the first callback (line 13). Maybe rephrase to `On 2 occasions you will have ...`

Also, the second callback is optional, "have to make" is then maybe a little bit too strong.

File: _sales_tracking/callbacks.md [hunk 2]
```diff
@@ -1,102 +1,21 @@
...
 
-  conn.request("POST", "/sales/status?source=backend_python&sale_id=SALE_ID&name=NAME&email=EMAIL&api_key=API_KEY")
+1. Upon checkout, when a sale [confirms]({{ site.baseurl }}/#sales_trackinggetting_started).
+2. Upon validation, when a sale [completes]({{ site.baseurl }}/#sales_trackinggetting_started) (optional).
```
Comment (harianus) [inline _sales_tracking/callbacks.md]: Maybe note here we can import sales based on CSV's where we have our sale id or their order id (if they did sent it in the callbacks / ping script).

**Comment by chrissvo** [comment]
Eh, if you read "callbacks" then it gives you 2 options, where the first option listed is the ping script. Right below it we state that the choice is theirs with the advantages/disadvantages listed very short. Getting started doesn't say anything about which sort of tracking, or am I missing something?

**Comment by chrissvo** [comment]
> The \'s looks a bit strange. Why not name: "Your new client's name",?

Cause the HTML fucks up, I'll change it to something different

**Comment by harianus** [comment]
image

**Comment by chrissvo** [comment]
This is branch, you're reading the old docs

**Comment by harianus** [comment]
I thought it was the Getting started section (which is not changed in this PR) but it is the Marketeers section. Never mind my comment then.

**Comment by chrissvo** [comment]
Here's how you can run this branch locally: https://github.com/mgmco/clients.membergetmember.co/blob/gh-pages/README.md

File: _sales_tracking/callbacks.md (hunk 2 above)
Comment (harianus) [inline _sales_tracking/callbacks.md]: We have /had the text already in the marketeer section:

>  Alternatively it is possible to import the validated sales through our CSV importer, as long as they are matched up with our `MGMco sale ID`!

File: _sales_tracking/callbacks.md (hunk 1 above)
Comment (chrissvo) [inline _sales_tracking/callbacks.md]: Yes!

File: _sales_tracking/callbacks.md (hunk 2 above)
Comment (chrissvo) [inline _sales_tracking/callbacks.md]: I'll a backlink and make it a bit clearer!

**Comment by harianus** [comment]
It is one page right? Could you create a .pdf of it?

**Comment by chrissvo** [comment]
[docs_#7.pdf](https://github.com/mgmco/clients.membergetmember.co/files/707580/docs_.7.pdf)

**Comment by harianus** [comment]
Awesome, thanks!

**Comment by chrissvo** [comment]
Looks quite good in PDF!

**Comment by mtimofiiv** [comment]
@chrissvo yes, the cookie can be specified by name via the `MGMbuyerInfo.cookie_name` property, or alternatively they can just pass the mgm sale ID via `MGMbuyerInfo.sale_id`

**Comment by chrissvo** [comment]
Okay, I'll add something on the `cookie_name`

**Comment by vireshnarain** [comment]
Can you change the marketing contact details from viresh@membergetmember.co to account@membergetmember.co? 

account@membergetmember.co gets to the inbox of @MartineHardeveld and moi

**Comment by harianus** [comment]
PR waiting for that @vireshnarain: https://github.com/mgmco/clients.membergetmember.co/pull/8
